tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -2,23 +2,30 @@
 Library             SeleniumLibrary
 Resource            ../../../Resources/Page/OCPLogin/OCPLogin.resource
 Resource            ../../../Resources/OCP.resource
-
+Resource            ../../../Resources/Page/ODH/AiApps/AiApps.resource
+Resource            src.yaml
 Test Setup          Dashboard Test Setup
 Test Teardown       Dashboard Test Teardown
 
 
 *** Variables ***
 ${pachyderm_container_name}     Pachyderm
+${pachyderm_operator_name}      Pachyderm
 ${pachyderm_appname}            pachyderm
 
 
 *** Test Cases ***
 Verify Pachyderm Can Be Installed Using OpenShift Console
-    [Tags]    Tier2    ODS-1137
+    [Tags]      Tier2
+    ...         ODS-1137    ODS-1138
     Check And Install Operator in Openshift    ${pachyderm_container_name}    ${pachyderm_appname}
+    Create Project      pachyderm
```

Comment:
I think lines 22-25 should be in a keyword called `Deploy Pachyderm Server`  (or similar). If you want to split ODS-1137 and ODS-1138 in two different tests you can follow the pattern in https://github.com/red-hat-data-services/ods-ci/blob/522735d488cd8c54c71e38b860f8917998ea2944/tests/Tests/600__ai_apps/640__rhoam/640__rhoam_installation.robot#L20